docs/INSTALLATION.md

```diff
@@ -1,14 +1,25 @@
-**Windows and OS X users**: Install [Node.js](http://nodejs.org/) [via package manager](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager). Windows users can use Visual Studio, see below.
+## Installing Node.js
 
-**Linux users**: [Joyent maintains a document][linstall] that details how to get Node.js installed for a wide range of distributions and package managers.
+There are at least two common options to run JavaScript code: the broswers and [Node.js](http://nodejs.org/). Here we are going to use Node.js to run our JavaScript code. Let's see how to install Node.js locally:
 
-[linstall]: https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager
+**Windows users**: find official installers on [Node.js downloads](https://nodejs.org/en/download/). Choose the LTS (long time support) version. It will install Node.js on your machine as well as `npm` (that stands for Node Package Manager), a tool that helps installing applications that run on top of Node.js.
```

Comment:
Ok, I direclty translated it. In my mind they mean more or less the same. I'll fix it.